pkg/acsengine/engine.go

```diff
@@ -329,7 +329,7 @@ func GetCloudSpecConfig(location string) AzureEnvironmentSpecConfig {
 	switch GetCloudTargetEnv(location) {
 	case azureChinaCloud:
 		return AzureChinaCloudSpec
-	//TODO - add cloud specs for germany and usgov
+		//TODO - add cloud specs for germany and usgov
```

Comment:
No idea, not my todo - just snagged it with `go fmt` can undo the change if we want